_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+What do we need to get started? We have a few options, we have Base Voicemeeter, Voicemeeter Banana, and Voicemeeter Potato. For the most part, I am skipping the setup of Base Voicemeeter. My reasons for this are that I don't believe it offers sufficient customization alone while Voicemeeter Banana is the same price of free and offers way more features. I'll be covering a feature comparison, just know I don't think it is worth trying to use Base Voicemeeter over Banana, especially for a stream setup.
+
+Now, if you would like to install the programs before I go over the very basic features and follow along, you can find them here: https://vb-audio.com/index.htm. I also highly recommend grabbing the Virtual Audio Cables as well. Virtual Audio Cables are useful to help with the routing of audio and will be used in some, if not all, of the setups. Installation might require a restart.
+
+### - Parts Explanations
+
+Voicemeeter? Banana? Potato? Marco? Polo? Right! The versions can be a little tricky at first but I will give a bit of an overview of the differences. Firstly, lets get a quick overview of the terms for things that are to come up.
+
+The basics are Hardware Inputs, Digital Inputs, and Hardware Outputs. You will likely notice that I will be refraining from going into depth on individual features in this article and mostly want to give a sum of the basics to get started. The extra features I would like to cover in a different article altogether.
+
+Lets go over what each of the basic pieces are. Hardware Inputs are typically something like a microphone, line input, or similar things. The Virtual Input is basically a digital audio device/cable. (I will get more into these in one of the setups and further explain them.) Lastly, we have Hardware Ouputs, these are things like headphones, speakers, or in the case of what we will be configuring for streaming in the setup part, a digital audio device/cable.
+
+[picture of a fully setup voicemeeter maybe?]
+
+All of these options can be confusing when first looking at them so what I will do is break down each portion per version of voicemeeter. The portions covered at this moment will be Hardware Outputs, then Digital Inputs, then Hardware Inputs. Why does it seem like I'll be going backwards from what I originally listed? Unlike what the program looks like, which is the order up above, this new order is based on being able to check to make sure everything is getting set up properly as you go. You need your outputs such as headphones for testing audio to make sure the inputs are set up properly. If we set all our inputs up and then our outputs, we would be blaring everything all together and testing would be more difficult.
```

Comment:
It could be helpful to rework from this paragraph up to the "Parts Explanations" header. If you are walking someone through the UI and actually teaching them to set up an audio route, mention that to them and state that we are setting up the audio output first to make sure that when we do get audio routed correctly through Voicemeeter that we will immediately start hearing it.

You may also want to write up a short list of steps to correctly set up the audio route as well. For example:

"In order to set up Voicemeeter Banana correctly for an audio route, we need to:

1. Set up an output device to headphones or speakers.
2. Connect an input device to an output device
3. Send audio into the input device.
"

One thing I got tripped up on as well was that the output of a program went into the input of a virtual cable, and the output of the cable needs to go into the "input device" of a specific Voicemeeter channel. This may need to be explicitly called out as well to really drill home that Outputs from programs go into Inputs in Voicemeeter. You don't connect an Input to an output or an input to an input.